style.css

```diff
@@ -0,0 +1,278 @@
+/* CSS Fiile for Davins Portfolio */
+
+
+/* -------------------------------------------------------------
+	Generic Body Stuff 
+---------------------------------------------------------------- */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	background-color:  hsl(204, 46%, 20%);
+	font-family: Verdana;
+	font-size: 90%;
+	
+}
+
+
+/* ---------------------------------------------------------------------------
+	Generic Classes for  Containers & Sections and Cursive Font for Sections
+---------------------------------------------------------------------------- */
+
+.mainContainerGrid {
+	/* nothing required here, only MQ later */
```

Comment:
if nothing required, please remove the selector

Reply:
removed